### PR TITLE
Update owners.json

### PR DESCRIPTION
**Pull request by DominikDary**
## High-level description

Updating owners

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [ ] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [ ] Included a test which will fail if code is reverted but test is not. If there is no test please explain.
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]

### CODE REVIEW

**Comment by DominikDary** [comment]
@mesosphere-mergebot request-review jgehrcke

File: owners.json [hunk 1]
```diff
@@ -3,22 +3,24 @@
...
   "Gilbert88": {"slack": "gilbert", "jira": "gilbert"},
   "gkleiman": {"slack": "gaston", "jira": "gaston"},
   "gpaul": {"slack": "gpaul", "jira": "gpaul"},
   "greggomann": {"slack": "greg", "jira": "greg"},
   "jgehrcke": {"slack": "jp", "jira": "jp"},
+  "jr0d": {"slack": "jared.rodriguez", "jira": "jaredrodriguez"},
   "jongiddy": {"slack": "jongiddy", "jira": "jonathangiddy"},
   "juliangieseke": {"slack": "Julian", "jira": "juliangieseke"},
   "meichstedt": {"slack": "matthias.eichstedt", "jira": "matthias.eichstedt"},
+  "mhrabovcin": {"slack": "martin.hrabovcin", "jira": "mhrabovcin.c"},
```
Comment (mhrabovcin) [inline owners.json]: slack = `martin`

**Comment by DominikDary** [comment]
@mesosphere-mergebot changelog-not-required Updating owners file.

**Comment by jongiddy** [comment]
@mesosphere-mergebot sync

**Pull request merged**